Libplanet.Extensions.Cocona/Commands/KeyCommand.cs

```diff
@@ -177,9 +239,16 @@ public KeyCommand()
                                   "as raw bytes not hexadecimal string or else. " +
                                   "If this option isn't given, it will print hexadecimal string " +
                                   "to stdout as default behaviour.")]
-            string? binaryOutput = null
+            string? binaryOutput = null,
+            [Option(Description = "Path to key store to use key from.")]
+            string? storePath = ""
         )
         {
+            if (storePath != null)
+            {
+                KeyStore = new Web3KeyStore(Path.GetFullPath(path));
+            }
```

Comment:
Though it introduced a new argument `string?` typed `storePath`, the default value seems like `""`. Maybe it will be always not `null`. What do you think about it?

Reply:
fixed as `string? storePath = null`.